app/models/episode_story_handler.rb

```diff
@@ -1,7 +1,7 @@
 require 'episode'
 
 class EpisodeStoryHandler
-  include PRXAccess
+  include PrxAccess
```

Comment:
Ok, the `PRX` inflection was not compatible with prx_auth-rails, and so I renamed the feeder classes to use `Prx`.
Found in testing default Rails 7 pre-loading with `zeitwerk`